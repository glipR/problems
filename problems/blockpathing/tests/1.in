1 1
14 14
2 2 7 7
4 11 11
