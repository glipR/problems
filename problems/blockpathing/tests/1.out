21.625015745078453
